app/javascript/dashboard/components/ui/DatePicker.vue

```diff
@@ -1,13 +1,15 @@
 <template>
-  <div class="woot-date-range-picker">
+  <div class="date-picker">
     <date-picker
-      :range="true"
+      :type="type"
```

Comment:
Can we keep date range and date picker as separate components to keep it simple.

Reply:
If a future case comes up, we have to always check the type is Array or date depending on functionality.